test/Microsoft.AspNetCore.Session.Tests/SessionTests.cs

```diff
@@ -105,8 +105,11 @@ public class SessionTests
                {
                    app.UseSession(new SessionOptions
```

Comment:
@Tratcher @davidfowl @ajcvickers @divega thoughts?  Can we get rid of all instances where we `UseXyz(new XyzOptions())` and consistently always use `UseXyz(o => )`